test/e2e/events_test.go

```diff
@@ -177,9 +177,14 @@ var _ = Describe("Podman events", func() {
 
 		// unix timestamp in 10 seconds
 		until := time.Now().Add(time.Second * 10).Unix()
+		untilT := time.Now().Add(time.Second * 9)
 		result := podmanTest.Podman([]string{"events", "--since", "30s", "--until", fmt.Sprint(until)})
 		result.Wait(11)
 		Expect(result).Should(Exit(0))
+		tEnd := time.Now()
+		outDur := tEnd.Sub(untilT)
+		diff := outDur.Seconds() > 0
+		Expect(diff).To(Equal(true))
```

Comment:
To test your change you have to call `--until 10s` and not use the used unix timestamp.